one_fm/one_fm/custom/job_applicant.json

```diff
@@ -1,5 +1,137 @@
 {
  "custom_fields": [
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 0,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2024-12-03 13:14:45.325999",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Job Applicant",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "custom_interview_feedback_rating",
+   "fieldtype": "Rating",
+   "hidden": 0,
+   "hide_border": 0,
+   "hide_days": 0,
+   "hide_seconds": 0,
+   "idx": 4,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_preview": 0,
+   "in_standard_filter": 0,
+   "insert_after": "custom_interview_feedback",
+   "is_system_generated": 0,
+   "is_virtual": 0,
+   "label": "Interview Feedback Rating",
+   "length": 0,
+   "link_filters": null,
+   "mandatory_depends_on": null,
+   "modified": "2024-12-03 13:14:45.325999",
+   "modified_by": "Administrator",
+   "module": null,
+   "name": "Job Applicant-custom_interview_feedback_rating",
+   "no_copy": 0,
+   "non_negative": 0,
+   "options": "5",
+   "owner": "Administrator",
+   "parent": null,
+   "parentfield": null,
+   "parenttype": null,
+   "permlevel": 0,
+   "precision": "",
+   "print_hide": 0,
+   "print_hide_if_no_value": 0,
+   "print_width": null,
+   "read_only": 1,
+   "read_only_depends_on": null,
+   "report_hide": 0,
+   "reqd": 0,
+   "search_index": 0,
+   "show_dashboard": 0,
+   "sort_options": 0,
+   "translatable": 0,
+   "unique": 0,
+   "width": null
+  },
+  {
+   "_assign": null,
+   "_comments": null,
+   "_liked_by": null,
+   "_user_tags": null,
+   "allow_in_quick_entry": 0,
+   "allow_on_submit": 0,
+   "bold": 0,
+   "collapsible": 1,
+   "collapsible_depends_on": null,
+   "columns": 0,
+   "creation": "2024-12-03 13:14:45.064791",
+   "default": null,
+   "depends_on": null,
+   "description": null,
+   "docstatus": 0,
+   "dt": "Job Applicant",
+   "fetch_from": null,
+   "fetch_if_empty": 0,
+   "fieldname": "custom_interview_feedback",
+   "fieldtype": "Section Break",
+   "hidden": 0,
+   "hide_border": 0,
+   "hide_days": 0,
+   "hide_seconds": 0,
+   "idx": 3,
+   "ignore_user_permissions": 0,
+   "ignore_xss_filter": 0,
+   "in_global_search": 0,
+   "in_list_view": 0,
+   "in_preview": 0,
+   "in_standard_filter": 0,
+   "insert_after": "interview_rounds",
+   "is_system_generated": 0,
+   "is_virtual": 0,
+   "label": "Interview Feedback",
+   "length": 0,
+   "link_filters": null,
+   "mandatory_depends_on": null,
+   "modified": "2024-12-03 13:14:45.064791",
+   "modified_by": "Administrator",
+   "module": null,
+   "name": "Job Applicant-custom_interview_feedback",
```

Comment:
In Job applicant there is an inbuilt summary section
![Screenshot 2024-12-03 at 9 45 28 PM](https://github.com/user-attachments/assets/63ee7e86-3e51-4218-8c81-cc44a4b87210)

Reply:
or rather, if i use the applicant rating, there wont be need for that anymore